.github/workflows/release.yml

```diff
@@ -3,6 +3,7 @@ name: Create Release Branch
 on:
   schedule:
     - cron: '0 10 * * 2'
+  workflow_dispatch:
```

Comment:
will it allow to execute pipeline manually?

Reply:
Yep, it will add "Run workflow" button to https://github.com/neondatabase/neon/actions/workflows/release.yml page.
And will allow running the workflow using gh cli: `gh workflow run release.yml`